### ISSUE TITLE
Getting error: Cannot destructor property 'name' of 'undefined' as it is 'undefined'

### ISSUE THREAD

**Issue by alcarcdr** [mtxr/vscode-sqltools#884]
In the SQL Console: Message output the following error happens with some basic MySQL DDL failing:
```
Cannot destructor property 'name' of 'undefined' as it is 'undefined'
````
The MySQL that is being executed is:

```
DROP TABLE IF EXISTS album;
CREATE TABLE album (
  id         INT AUTO_INCREMENT NOT NULL,
  title      VARCHAR(128) NOT NULL,
  artist     VARCHAR(255) NOT NULL,
  price      DECIMAL(5,2) NOT NULL,
  PRIMARY KEY (`id`)
);
``` 
(This works fine if executed using MySQL CLI.)

**To Reproduce**
Steps to reproduce the behaviour:
1. Create MySQL Schema DB and open connection to it
2. Paste the MySQL SQL code above and execute it
3. See that it fails with error mentioned above

**Expected behaviour**
Expected the Script to execute and a new DB Table called 'album' to be created. If the Table already existed it would be dropped and new one created. 
No error was expected.

**Desktop (please complete the following information):**
 - SQLTools Version v0.23.0 
 - VSCode Version: 1.62.3
 - OS: Windows 11 (up to date)
 - Driver: 
     - [ ] MySQL/MariaDB
 - Database version:  MySQL Community Server v8.0.18

**Additional context**

After doing some searching around I found that the error message is most likely a Typescript/Javascript one. My project is not using this as its a basic Go project, actually I don't have much in the way of code in it yet and the code I have is working fine.

**Comment by amgilder** [comment]
If it helps, I can add some further experience to the original report. I too am encountering this error when interacting with MySQL (MacOS 12.1, VS Code 1.63.2, SQLTools 0.23.0, SQLTools MySQL/MariaDB 0.2.0, MySQL Community Server 8.0.27).  In my experience, the error arises whenever there are multiple SQL statements in the SQL file being run. With only a single SQL statement in the file, the error does not arise. Also, in every case I've so far encountered where it was not the output of the statements but rather their side effects that concerned me, I have found that the side effects are actually occurring. That is, if I give it several CREATE TABLE statements in a single file, all of the tables will be created as expected, despite the error. However, if the several statements include one or more SELECT statements, I am not able to see their result sets due to the error. With any luck, that will help you narrow things down more efficiently.

**Comment by gjsjohnmurray** [comment]
See also #864

**Comment by gjsjohnmurray** [comment]
Fixed in the 0.4.0 MySQL driver which was published today.
